application/schemas/user_schema.py

```diff
@@ -0,0 +1,17 @@
+""" Defines schema for database Team objects """
+from marshmallow import fields
+
+from ..base import BaseSchema, output_decorator
+
+
+class UserSchema(BaseSchema):
+    """ Schema for importing and exporting Team objects """
+    id_ = fields.UUID(missing=None)
```

Comment:
id_ is actually saved simply as a string. I'm not sure if that might mess with the schema later. You might wanna consider just making this a string with a `validate_uuid` hook (same in all other model schemas then though)